src/cider/nrepl/middleware/debug.clj

```diff
@@ -502,7 +502,8 @@ this map (identified by a key), and will `dissoc` it afterwards."}
   [coor val locals STATE__]
   (if-let [first-coor @(:session-id STATE__)]
     (when (= first-coor coor)
-      (reset! (:skip STATE__) false))
+      (reset! (:skip STATE__) false)
+      (skip-breaks! false))
```

Comment:
Can you also add some explanatory comments here, so this part of the code is easier to understand for future readers?

Reply:
Sure, I'll add comment.